subsys/net/l2/ethernet/arp.c

```diff
@@ -339,15 +339,23 @@ struct net_pkt *net_arp_prepare(struct net_pkt *pkt,
 {
 	struct arp_entry *entry;
 	struct in_addr *addr;
+	bool is_ipv4_ll_used = false;
```

Comment:
just to stay consistent throughout net stack: move the initialized variable at first position in the list.

Reply:
Done